.claude/agents/perf-test-sync.md

```
@@ -0,0 +1,376 @@
+    input_length: <benchmark.input_length from source disagg>
+    output_length: <benchmark.output_length from source disagg>
+    dataset_file: <dataset_file>
+```
+
+Notes:
+- NVIDIA copyright header on every new file (year = current).
+- Use placeholders (`<partition>`, `<container_image>`, etc.) consistent with existing QA YAMLs — do not hardcode real values.
+- If multiple disagg cases share the same canonical ctx key but differ on `benchmark.input_length`/`output_length`, generate one agg YAML per (ctx, isl/osl) pair.
+
+c. **Test list entry**: append to the `# aggregated multi-node` → `# GB200 + GB300 supported cases` subsection of `llm_perf_multinode.txt` using the **ctx_only form (no `_upload`)**:
+   ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)